Packs/GenericWebhook/Integrations/GenericWebhook/README.md

```diff
@@ -10,11 +10,12 @@ The Generic Webhook integration is a long-running integration. For more informat
 
 | **Parameter** | **Description** | **Required** |
 | --- | --- | --- |
-| Listen Port | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. <br>Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use.                           | True |
+| Listen Port<br><br>Note: this field will not appear in Cortex XSOAR 8 and Cortex XSIAM unless using an custom engine | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. <br>Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use.                           | True - unless using a custom engine (in Cortex XSOAR 8 and Cortex XSIAM only). |
```

Comment:
Listen Port<br><br>Note: this field will not appear in Cortex XSOAR 8 and Cortex XSIAM unless using a custom engine.